ducktape/services/service.py

```diff
@@ -25,18 +25,24 @@
 # that there won't be resource conflicts: the cluster tests are being run on
 # should be large enough to use one instance per service instance.
 
-from ducktape.logger import Logger
 
-
-class Service(Logger):
-    def __init__(self, cluster, num_nodes):
-        self.num_nodes = num_nodes
-        self.cluster = cluster
+class Service(object):
+    def __init__(self, service_context):
+        """
+        :type service_context: ServiceContext
+        """
+        self.num_nodes = service_context.num_nodes
+        self.cluster = service_context.cluster
+        self.logger = service_context.logger
 
     def start(self):
         """Start the service running in the background."""
         self.nodes = self.cluster.request(self.num_nodes)
         for idx, node in enumerate(self.nodes, 1):
+
+            # Remote accounts utilities should log where this service logs
+            node.account.set_logger(self.logger)
```

Comment:
Should there be a corresponding call in stop() or some other method that clears out the logger. That would offer some opportunity for doing error checking by testing whether the logger was already set (which could signal some race or framework error when we start doing parallel tests, or some user error if they aren't cleaning up properly; in the latter case, we might actually want set_logger() to pass in extra information so that if we catch an error we can also log which services/tests are conflicting).